### ISSUE TITLE
Auto close when using optifine 1.18.2

### ISSUE THREAD

**Issue by kedrick021** [PojavLauncherTeam/PojavLauncher#3520]
### Describe the bug

Forge optifine 1.18.2
Auto close when create world Minecraft 

### The log file and images/videos

None

### Steps To Reproduce

```markdown
1. Using forge optifine 1.18.2
2. Start pojavlauncher
3. Create world 
4. Close game
```

### Expected Behavior

Working only forge
When I add optifine
Auto close on Minecraft create world

### Platform

```markdown
- Device model: umidigi bison
- CPU architecture: 64
- Android version: 10
- PojavLauncher version: latest on Google play store
```

### Anything else?

Plss fix

**Comment by GaucovyAero** [comment]
make sure you have render regions set to OFF in optifine performance tab

**Comment by kedrick021** [comment]
There off

**Comment by mchnjt** [comment]
![924504368890392586](https://user-images.githubusercontent.com/83752061/187030898-201c010a-dcd3-430f-8f1f-b249c84b358c.png)

**Comment by kedrick021** [comment]
Am using forge 40.1.73 latest not working if I add optifine

**Comment by GaucovyAero** [comment]
> Am using forge 40.1.73 latest not working if I add optifine

send latestlog.txt then

**Comment by kedrick021** [comment]
[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/9439269/latestlog.txt)
When create Minecraft world loading 0%then crash

**Comment by GaucovyAero** [comment]
> [latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/9439269/latestlog.txt) When create Minecraft world loading 0%then crash

```java.lang.NoSuchMethodError: 'void net.minecraft.server.level.DistanceManager.addRegionTicket(net.minecraft.server.level.TicketType, net.minecraft.world.level.ChunkPos, int, java.lang.Object, boolean)'```

some mod is incompatible, you will have to figure that out

**Comment by kedrick021** [comment]
I only using optifine mod

**Comment by mchnjt** [comment]
when the

**Comment by GaucovyAero** [comment]
> I only using optifine mod

Oh wait, right, I was totally diagnosing this wrong. Make sure to disable Render Regions in Performance settings in OptiFine.

**Comment by kedrick021** [comment]
> > I only using optifine mod
> 
> Oh wait, right, I was totally diagnosing this wrong. Make sure to disable Render Regions in Performance settings in OptiFine.

The render region is off already

**Comment by GaucovyAero** [comment]
> > > I only using optifine mod
> > 
> > 
> > Oh wait, right, I was totally diagnosing this wrong. Make sure to disable Render Regions in Performance settings in OptiFine.
> 
> The render region is off already

well, it shouldn't crash then

**Comment by kedrick021** [comment]
> > > > I only using optifine mod
> > > 
> > > 
> > > Oh wait, right, I was totally diagnosing this wrong. Make sure to disable Render Regions in Performance settings in OptiFine.
> > 
> > 
> > The render region is off already
> 
> well, it shouldn't crash then

Why don't you try to play 1.18.2 forge optifine you know what am trying to say

**Comment by GaucovyAero** [comment]
> > > > > I only using optifine mod
> > > > 
> > > > 
> > > > Oh wait, right, I was totally diagnosing this wrong. Make sure to disable Render Regions in Performance settings in OptiFine.
> > > 
> > > 
> > > The render region is off already
> > 
> > 
> > well, it shouldn't crash then
> 
> Why don't you try to play 1.18.2 forge optifine you know what am trying to say

Alright, ima do it now.

**Comment by GaucovyAero** [comment]
I mean, it's working fine for me. Make sure to use Forge 40.1.0 instead of latest (you are using Forge 40.1.73, use the older one)

![Screenshot_20220903-211356_PojavLauncher_Minecraft__Java_Edition_for_Android](https://user-images.githubusercontent.com/85581164/188285268-cc4a0426-32af-4f00-b05e-bc0c8f39b020.png)

**Comment by kedrick021** [comment]
> I mean, it's working fine for me. Make sure to use Forge 40.1.0 instead of latest (you are using Forge 40.1.73, use the older one)
> 
> ![Screenshot_20220903-211356_PojavLauncher_Minecraft__Java_Edition_for_Android](https://user-images.githubusercontent.com/85581164/188285268-cc4a0426-32af-4f00-b05e-bc0c8f39b020.png)

Forge 40.1.60 - 73 are not working I want the latest

**Comment by GaucovyAero** [comment]
> > I mean, it's working fine for me. Make sure to use Forge 40.1.0 instead of latest (you are using Forge 40.1.73, use the older one)
> > ![Screenshot_20220903-211356_PojavLauncher_Minecraft__Java_Edition_for_Android](https://user-images.githubusercontent.com/85581164/188285268-cc4a0426-32af-4f00-b05e-bc0c8f39b020.png)
> 
> Forge 40.1.60 - 73 are not working I want the latest

well, it doesn't even work on PC, so i guess you can't do anything about it.

**Comment by GaucovyAero** [comment]
Stale issue and fixed. Closing issue.

**Comment by kedrick021** [comment]
> Stale issue and fixed. Closing issue.

Really it's really fixed I can play 1.18.2 latest?
